README.rst

```diff
@@ -66,10 +66,12 @@ Installation
 ------------
 
 These instructions detail how to configure your operating system so that
-CodeJail can execute Python code safely.  You can run CodeJail without these
-steps, and you will have an unsafe CodeJail.  This is fine for developers'
-machines who are unconcerned with security, and simplifies the integration of
-CodeJail into your project.
+CodeJail can execute Python code safely. However, it is also possible to set
```

Comment:
Not sure of all the places to put this, but I like sentences like "Do not do use this in production systems.", or something of that nature. I think it is loud and clear and gets right to the main point.

Reply:
Made some updates, including removing/rewording a section farther up on the same topic.